cc/main.cc

```diff
@@ -465,46 +468,232 @@ class SelfPlayer {
   uint64_t flags_timestamp_ = 0;
 };
 
+class Evaluator {
```

Comment:
this seems like something that would be an ideal candidate to break out of main.cc

Reply:
Yes. Tom requested that I hold off chopping up main.cc for now. Will do in a separate CL.